spec/index.bs

```diff
@@ -5131,70 +5144,114 @@ enum GPUStoreOp {
 <dl dfn-type=method dfn-for=GPURenderEncoderBase>
     : <dfn>setPipeline(pipeline)</dfn>
     ::
+        Sets the current {{GPURenderPipeline}}.
 
         <div algorithm="GPURenderEncoderBase.setPipeline">
             **Called on:** {{GPURenderEncoderBase}} this.
 
             **Arguments:**
             <pre class=argumentdef for="GPURenderEncoderBase/setPipeline(pipeline)">
-                pipeline:
+                |pipeline|: The render pipeline to use for subsequent drawing commands.
             </pre>
 
             **Returns:** void
 
-            Issue: Describe {{GPURenderEncoderBase/setPipeline()}} algorithm steps.
+            Issue the following steps on the [=Device timeline=] of |this|:
+            <div class=device-timeline>
+                1. If any of the following conditions are unsatisfied, generate a
+                    {{GPUValidationError}} error and stop.
```

Comment:
This should make the encoder invalid, instead of generating a GPUValidationError; same elsewhere